### PR TITLE
Parmmg interface cleanup

### PR DESCRIPTION
**Pull request by lcirrottola**
Trying to get global numbering on all nodes with the new API.

### CODE REVIEW

File: applications/MeshingApplication/custom_utilities/pmmg_utilities.cpp [hunk 1]
```diff
@@ -962,6 +971,10 @@ void ParMmgUtilities<PMMGLibrary::PMMG3D>::PMMGLibCallMetric(Parameters Configur
     if ( PMMG_Set_iparameter(mParMmgMesh,PMMG_IPARAM_APImode, static_cast<int>(PMMG_APIDISTRIB_nodes)) != 1 )
         KRATOS_ERROR << "Unable to set initialize parallel library through interface faces or nodes" << std::endl;
 
+    // Node global numbering: Forced to "1" all the time
+    if ( PMMG_Set_iparameter(mParMmgMesh,PMMG_IPARAM_nodeGloNum,1) != 1 )
```
Comment (marcnunezc) [inline applications/MeshingApplication/custom_utilities/pmmg_utilities.cpp:975]: What does the "1" mean? Is it the starting global Id?

Comment (marcnunezc) [reply applications/MeshingApplication/custom_utilities/pmmg_utilities.cpp:975]: Ok , just saw the comments on the header file for the function InitNodeGloNumParameter() 😅 

If I understand correctly it is a way to toggle the global vs local id numbering, right?

**Review by marcnunezc — APPROVED** [review]
Thanks Luca! Aside from my questions below I don't see big potential issues!

File: applications/MeshingApplication/custom_utilities/pmmg_utilities.cpp (hunk 1 above)
Comment (lcirrottola) [reply applications/MeshingApplication/custom_utilities/pmmg_utilities.cpp:975]: Yes, it is just a switch to toggle it! (1==on, 0==off).
Numbering instead always starts from 1.

**Pull request merged**